corehq/form_processor/migrations/0078_blobmeta_migrated_check.py

```diff
@@ -39,27 +36,19 @@ def _assert_blobmetas_migrated(apps, schema_editor):
         return
 
     if num_attachments < 10000:
-        try:
-            call_command(
-                "run_sql",
-                "simple_move_form_attachments_to_blobmeta",
-                dbname=schema_editor.connection.alias,
-            )
-            migrated = get_num_attachments(schema_editor.connection) == 0
-            if not migrated:
-                print("Automatic migration failed")
-                migrated = False
-        except Exception:
-            traceback.print_exc()
+        run_management_command_or_exit(
+            "run_sql", "simple_move_form_attachments_to_blobmeta",
+            dbname=schema_editor.connection.alias,
+            custom_message=BLOBMETAS_NOT_MIGRATED_ERROR
+        )
+        remaining = get_num_attachments(schema_editor.connection)
+        if remaining != 0:
+            print(f"Automatic migration failed, {remaining} items remain to migrate.")
+            sys.exit(1)
     else:
         print("Found %s attachments." % num_attachments)
         print("Too many to migrate automatically.")
```

Comment:
This branch is broken too.

```suggestion
        print("Too many to migrate automatically.")
        print("")
        print(BLOBMETAS_NOT_MIGRATED_ERROR)
        sys.exit(1)
```